ga4gh/datarepo.py

```diff
@@ -110,15 +110,17 @@ def getNumReferenceSets(self):
         """
         return len(self._referenceSetIds)
 
-    def getOntology(self, name):
+    def getOntologyTermMapByName(self, name):
         """
-        Returns an ontology map by name
+        Returns an ontologyTermMap by name
         """
-        return self._ontologyNameMap.get(name, None)
+        if name not in self._ontologyNameMap:
+            raise exceptions.OntologyNameNotFoundException(name)
```

Comment:
This exception is thrown when we can't find the OntologyTermMap itself (when, say, ``sequence_ontology`` isn't there), not when a term mapping failure occurs. Term mapping failures map silently to the empty string as before, which is probably all we can do (although, I think these failures should be logged; see #1113).